deps/redis/util.h

```diff
@@ -18,5 +18,7 @@ int ll2string(char *s, size_t len, long long value);
 int ull2string(char *s, size_t len, unsigned long long value);
 int lpStringToInt64(const char *s, unsigned long slen, int64_t *value);
 unsigned int getEnvVar(const char* varName, unsigned int defaultVal);
+double zzlStrtod(unsigned char *vstr, unsigned int vlen);
+int d2string(char *buf, size_t len, double value);
```

Comment:
Fyi, these two functions are used for double <-> string conversion in redis.

`zzlStrtod()` is used when you get double out of sorted set. Although it is a small function, I just copied it exactly just as a reminder that this was copied. One issue is it doesn't do error handling internally, I guess this is not a big problem as this is how it works right now in redis.